daemon/container_operations_windows.go

```diff
@@ -44,7 +44,14 @@ func (daemon *Daemon) setupConfigDir(c *container.Container) (setupErr error) {
 	for _, configRef := range c.ConfigReferences {
 		// TODO (ehazlett): use type switch when more are supported
 		if configRef.File == nil {
-			logrus.Error("config target type is not a file target")
+			// Runtime configs are not mounted into the container, but they're
+			// a valid type of config so we should not error when we encounter
+			// one.
+			if configRef.Runtime == nil {
```

Comment:
Maybe move this up  so it's `if configRef.File == nil && configRef.Runtime == nil`?

Reply:
Nah, because in either case you're going to `continue`, so you'd end up writing

```go
if configRef.File == nil && configRef.Runtime == nil {
    logrus.Error("config target type is not a file or runtime target")
    continue
}
if configRef.Runtime != nil {
    continue
}
```

Reply:
Ah I see.